translations/en-us.yaml

```diff
@@ -2738,6 +2738,15 @@ machine:
     sshPassword:
       label: SSH Password
       placeholder: Set Instance SSH Password, Blank for Auto set
+    errors:
+      nameTooShort: Name should have more than two characters.
+      nameNotValid: Name doesn't match the rule of Aliyun ECS hostname rule.
+      nameTooLong: The name cannot have more than thirty characters.
+      accessKeyRequired: Access key is required.
+      accessSecretRequired: Access Secret is required.
+      sshPasswordLengthNotValid: SSH Password should have at least eight characters and less than thirty characters.
+      sshPasswordInvalidCharacter: SSH Password has invalid characters.
+      sshPasswordFormatError: SSH Password must contain at least three of uppercase letters, lowercase letters, numbers and special characters.
```

Comment:
`SSH password must contain at least three out of four kinds of following characters: uppercase letter, lowercase letters, numbers, and special characters.`